tests/integration/targets/vmware_ops_deploy_ovf_test/runme.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+source ../init.sh
+
+# Extract the ansible_tags from integration_config.yml
+ANSIBLE_TAGS=$(awk '/ansible_tags/ {print $2}' ../../integration_config.yml)
+
+# Check if the ANSIBLE_TAGS variable is set
+if [[ -n "$ANSIBLE_TAGS" ]]; then
```

Comment:
I didn't notice it before, but maybe worth to add default `integration-ci` if not ANSIBLE_TAGS not set? Like:
ANSIBLE_TAGS=$(awk '/ansible_tags/ {print $2}' ../../integration_config.yml)
ANSIBLE_TAGS={$ANSIBLE_TAGS:integration-ci}